### DISCUSSION TITLE
how to get real sql before exec

### DISCUSSION THREAD

**Discussion by baoyachi** [SeaQL/sea-orm#154] [Q&A] [answered]
Like 'diesel',the crate support method debug sql with method:[debug_query](https://docs.rs/diesel/1.4.7/diesel/fn.debug_query.html#) before exec .

Does sea-orm support? It's very useful function with debug.

**Comment by tyt2y3** [comment, marked as answer]
If you want to globally print every SQL statement, we have this feature but we haven't have written docs yet.

There are two ways
1) use the 'debug-print' feature
2) use the 'env-logger' crate

See #114

**Comment by billy1624** [comment]
Hi @baoyachi, welcome!

Does this help? https://www.sea-ql.org/SeaORM/docs/basic-crud/raw-sql#get-raw-sql-query

**Comment by baoyachi** [comment]
> Hi @baoyachi, welcome!
> 
> Does this help? https://www.sea-ql.org/SeaORM/docs/basic-crud/raw-sql#get-raw-sql-query

Yeah,but I don't what cake means in `sea-orm`.

**Comment by billy1624** [comment]
It's the name of an entity.
https://www.sea-ql.org/SeaORM/docs/generate-entity/entity-structure/

**Comment by baoyachi** [comment]
Some suggestion: the doc may be reference some examples link be better. 
The code may take time to read, lack context references, or be a better example to run

**Comment by tyt2y3** [comment]
So... you question got answered?

**Comment by baoyachi** [comment]
Done
